cli/src/test/resources/ide-projects/jmc/_ide/urls/java/java/17.0.6/linux_x64.urls

```diff
@@ -0,0 +1 @@
+http://localhost:1112/installTest/linux
```

Comment:
It seems you copy & pasted the `basic` project to `jmc`.
However, now with your changes only very few of the files you copied are really needed.
IMHO we should avoid duplicating lots of files for every commandlet that are never needed but instead start from scratch with the minimum required resources for the test. WDYT?